### ISSUE TITLE
Cannot edit or remove video

### ISSUE THREAD

**Issue by oleteacher** [fonini/ckeditor-youtube-plugin#75]
First, excellent job and thank you for sharing this plugin. Love the features.

Maybe I am missing it but cannot figure out how to edit prior embedded video or even remove?

If I try to highlight and use backspace and/or delete to remove video, only text around the video is removed.

Set config.allowedContent = true;

Running CKEditor 4.9

Cheers!

**Comment by hung-doan** [comment]
same here, not able to edit or remove the embedded video :(

**Comment by Shamaran** [comment]
Yeah, same here, Paste from URL option not working on edit,is removed the content(iframe) when replacing in the ckeditor and on save is saved without video

**Comment by oleteacher** [comment]
Looks like the plugin development is dead :( Shame, it was a good one...

**Comment by Shamaran** [comment]
yeah,but I did by allowing config.alloweContent=true;
